{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_drf_views.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.test import RequestFactory
+
+from {{ cookiecutter.project_slug }}.users.models import User
+from {{ cookiecutter.project_slug }}.users.api.views import UserViewSet
+from {{ cookiecutter.project_slug }}.users.api.serializers import UserSerializer
+
+pytestmark = pytest.mark.django_db
+
+
+class TestUserViewSet:
+    def test_get_queryset(self, user: User, rf: RequestFactory):
+        view = UserViewSet()
+        request = rf.get("/fake-url/")
+        request.user = user
+
+        view.request = request
+
+        assert user in view.get_queryset()
+
+    def test_me(self, user: User, rf: RequestFactory):
+        view = UserViewSet()
+        request = rf.get("/fake-url/")
+        request.user = user
+
+        view.request = request
+
+        response = view.me(request)
+
+        assert response.data == UserSerializer(user, context={"request": request}).data
```

Comment:
Ideally, this would not use the serializer to produce the expected data, can we compare to a dictionary instead? Something along these lines:

```suggestion
        assert response.data == {
            "username": user.username, 
            "email": user.email, 
            "name": user.name, 
            "url": f"/api/users/{user.username}/",
        }
```

Reply:
OK. I agree with your idea. Even to me, this code is more explicit to check result.